### ISSUE TITLE
Failed to precompile Bridge

### ISSUE THREAD

**Issue by ASaragga** [mschauer/Bridge.jl#42]
I am getting the following error when trying to precompile Bridge (Julia v1.1 on Mac)
```julia> using Bridge
[ Info: Precompiling Bridge [2d3116d5-4b8f-5680-861c-71f149790274]
ERROR: LoadError: LoadError: UndefVarError: Zip2 not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at ./sysimg.jl:13
 [2] top-level scope at none:0
 [3] include at ./boot.jl:326 [inlined]
 [4] include_relative(::Module, ::String) at ./loading.jl:1038
 [5] include at ./sysimg.jl:29 [inlined]
 [6] include(::String) at /Users/antonio/.julia/packages/Bridge/s3VsI/src/Bridge.jl:1
 [7] top-level scope at none:0
 [8] include at ./boot.jl:326 [inlined]
 [9] include_relative(::Module, ::String) at ./loading.jl:1038
 [10] include(::Module, ::String) at ./sysimg.jl:29
 [11] top-level scope at none:2
 [12] eval at ./boot.jl:328 [inlined]
 [13] eval(::Expr) at ./client.jl:404
 [14] top-level scope at ./none:3
in expression starting at /Users/antonio/.julia/packages/Bridge/s3VsI/src/types.jl:127
in expression starting at /Users/antonio/.julia/packages/Bridge/s3VsI/src/Bridge.jl:101
ERROR: Failed to precompile Bridge [2d3116d5-4b8f-5680-861c-71f149790274] to /Users/antonio/.julia/compiled/v1.1/Bridge/cfWTr.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1197
 [3] _require(::Base.PkgId) at ./loading.jl:960
 [4] require(::Base.PkgId) at ./loading.jl:858
 [5] require(::Module, ::Symbol) at ./loading.jl:853```

**Comment by mschauer** [comment]
This was fixend on

**Comment by mschauer** [comment]
This was fixed on master by #40 I believe, can you try
```
pkg> add Bridge#master
```
at the Pkg REPL prompt (hit `]` at the REPL)?

**Comment by ASaragga** [comment]
Yes, it is fine now on master. Thank you very much
